src/kia_hyundai_api/us_kia.py

```diff
@@ -291,11 +340,17 @@ async def start_climate(
                     "unit": 1,
                     "value": str(set_temp),
                 },
+                "heatVentSeat": {
```

Comment:
Given the source in the other integration you referenced this heatVentSeat should only be added if any of the seat parameters are not none